pootle/apps/pootle_statistics/migrations/0012_remove_create_subs.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.5 on 2017-02-17 17:54
+from __future__ import unicode_literals
+
+import logging
+import time
+
+from django.db import migrations
+
+
+logger = logging.getLogger(__name__)
+
+
+def remove_create_subs(apps, schema_editor):
+    subs = apps.get_model("pootle_statistics.Submission").objects.all()
+    to_delete = subs.filter(type=10)
```

Comment:
I would a comment telling that this is the value for `SubmissionTypes.UNIT_CREATE`.